src/libraries/System.Net.Security/src/System/Net/Security/SslStreamPal.Android.cs

```diff
@@ -0,0 +1,179 @@
+        }
+
+        public static SafeFreeCredentials AcquireCredentialsHandle(
+            SslStreamCertificateContext? certificateContext,
+            SslProtocols protocols,
+            EncryptionPolicy policy,
+            bool isServer)
+        {
+            return new SafeFreeSslCredentials(certificateContext, protocols, policy);
+        }
+
+        internal static byte[]? GetNegotiatedApplicationProtocol(SafeDeleteContext? context)
+        {
+            if (context == null)
+                return null;
+
+            throw new NotImplementedException(nameof(GetNegotiatedApplicationProtocol));
+        }
+
+        public static SecurityStatusPal EncryptMessage(
+            SafeDeleteContext securityContext,
+            ReadOnlyMemory<byte> input,
+            int headerSize,
+            int trailerSize,
+            ref byte[] output,
+            out int resultSize)
+        {
+            resultSize = 0;
+            Debug.Assert(input.Length > 0, $"{nameof(input.Length)} > 0 since {nameof(CanEncryptEmptyMessage)} is false");
+
+            throw new NotImplementedException(nameof(EncryptMessage));
+        }
+
+        public static SecurityStatusPal DecryptMessage(
+            SafeDeleteContext securityContext,
+            byte[] buffer,
+            ref int offset,
+            ref int count)
+        {
+            throw new NotImplementedException(nameof(DecryptMessage));
+        }
+
+        public static ChannelBinding? QueryContextChannelBinding(
+            SafeDeleteContext securityContext,
+            ChannelBindingKind attribute)
+        {
+            if (attribute == ChannelBindingKind.Endpoint)
+                return EndpointChannelBindingToken.Build(securityContext);
+
+            throw new NotImplementedException(nameof(QueryContextChannelBinding));
+        }
+
+        public static void QueryContextStreamSizes(
+            SafeDeleteContext? securityContext,
+            out StreamSizes streamSizes)
+        {
+            streamSizes = StreamSizes.Default;
+        }
+
+        public static void QueryContextConnectionInfo(
+            SafeDeleteContext securityContext,
+            out SslConnectionInfo connectionInfo)
+        {
+            connectionInfo = new SslConnectionInfo(((SafeDeleteSslContext)securityContext).SslContext);
+        }
+
+        private static SecurityStatusPal HandshakeInternal(
+            SafeFreeCredentials credential,
+            ref SafeDeleteSslContext? context,
+            ReadOnlySpan<byte> inputBuffer,
+            ref byte[]? outputBuffer,
+            SslAuthenticationOptions sslAuthenticationOptions)
+        {
+            Debug.Assert(!credential.IsInvalid);
+
+            try
+            {
+                SafeDeleteSslContext? sslContext = ((SafeDeleteSslContext?)context);
+
+                if ((null == context) || context.IsInvalid)
```

Comment:
I, personally, don't have a style rule here; but I think that the repository in general doesn't use left-literal-equals.  There were a couple of these.